### DISCUSSION TITLE
fedora

### DISCUSSION THREAD

**Discussion by eamat** [ffcrus/garlic#2] [General]
попробовал в fedora36 звук есть - значит скрипт запускается, а собственно изменение расклдки выделенного текста не работает

**Comment by ffcrus** [comment]
Жаль. Значит на Fedora36 не работает. Попробуйте выполнить: `echo $XDG_SESSION_TYPE`. Если вернёт не X11, то работать должно, будем разбираться. Ещё можно попробовать выполнить `loginctl show-session $(loginctl | grep $(whoami) | awk '{print $1}') -p Type` - тоже должно отдать X11.
